sdks/python/src/opik/runtime_config.py

```diff
@@ -0,0 +1,90 @@
+"""
+Runtime configuration for Opik tracing.
+
+This module enables dynamic enabling or disabling of Opik tracing at runtime
+without requiring application restarts or code edits. It acts as a thin,
+thread-safe wrapper around a global flag, falling back to the static
+configuration (`OpikConfig.track_disable`) when no runtime override is set.
+"""
+
+from __future__ import annotations
+
+import threading
+from typing import Optional
+
+
+class _RuntimeConfig:
```

Comment:
Let's move it to `opik.decorator` namespace and make the name more verbose - TrackingRuntimeConfig.